### DISCUSSION TITLE
Next suggestion on vscode with Spanish keyboard

### DISCUSSION THREAD

**Discussion by nosolosoft** [community/community#13848] [Copilot Conversations]
Hi,

I can't get next suggestion on vscode with Spanish keyboard. Keys to press should be `Alt + ]` or `Alt + [` but it doesn't works...

Anybody knows how to do it?

Regards,
Manu.

**Comment by D-Mx** [comment]
In my case I set my own shortcuts following this discussion https://github.com/community/community/discussions/6790. Remember to also set the `When` field.
